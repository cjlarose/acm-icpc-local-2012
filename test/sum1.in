4
5 15 
10678 21920
-782 678
9283 -9283
