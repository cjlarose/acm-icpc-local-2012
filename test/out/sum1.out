20
32598
-104
0
